integration/build.gradle

```diff
@@ -3,6 +3,29 @@ repositories {
     maven { url 'http://repository.jboss.org/nexus/content/groups/public' }
 }
 
+project.configurations {
+    integration {
+        extendsFrom project.configurations.testRuntime
+        description = 'Dependencies for integration tests'
+        transitive = true
+        visible = true
+    }
+    alpnboot
+}
+
+def getAlpnVersion() {
```

Comment:
The code refers to http://www.eclipse.org/jetty/documentation/9.2.8.v20150217/alpn-chapter.html#alpn-versions
ALPN is responsible for HTTP/2 protocol negotiation over TLS, hence its obligatory if we want an HTTP/2 integration test. As far as I'm concerned, Travis works on Java 1.8.0_31, so we could get rid of this code and stick to newest ALPN, though this may change in the future.